src/pages/contact_information.tsx

```diff
@@ -0,0 +1,107 @@
+import { Card, Image, Text, Button, Group, CardSection } from "@mantine/core";
+
+interface ContactCardProps {
+	title: string;
+	operationTitle: string;
+	operationTime: string;
+	imgUrl: string;
+	email: string;
+}
+
+const imageUrl = "/assets/images/logos/registrar.png";
+
+const contactCards = [
+	{
+		imgUrl: imageUrl,
+		title: "Academic Support",
+		operationTitle: "Operation Hours:",
+		operationTime: "Mon-Fri 08:30 - 17.30",
+		email: "ug_academic@stamford.edu",
+	},
+	{
+		imgUrl: imageUrl,
+		title: "Registrar",
+		operationTitle: "Operation Hours:",
+		operationTime: "Mon-Fri 08:30 - 17.30",
+		email: "registrar.bkk@stamford.edu",
+	},
+	{
+		imgUrl: imageUrl,
+		title: "Library",
+		operationTitle: "Operation Hours:",
+		operationTime: "Mon-Fri 08:30 - 17.30",
+		email: "library@stamford.edu",
+	},
+	{
+		imgUrl: imageUrl,
+		title: "Finance",
+		operationTitle: "Operation Hours:",
+		operationTime: "Mon-Fri 08:30 - 17.30",
+		email: "cashier_rm9@stamford.edu",
+	},
+	{
+		imgUrl: imageUrl,
+		title: "IT Support",
+		operationTitle: "Operation Hours:",
+		operationTime: "Mon-Fri 08:30 - 17.30",
+		email: "support@stamford.edu",
+	},
+	{
+		imgUrl: imageUrl,
+		title: "Student Service",
+		operationTitle: "Operation Hours:",
+		operationTime: "Mon-Fri 08:30 - 17.30",
+		email: "international-service@stamford.edu",
+	},
+];
+
+const ContactCard = ({ title, operationTime, operationTitle, imgUrl, email }: ContactCardProps) => (
+	<Card className="mx-auto max-w-[17rem] rounded-lg p-10 md:p-10 lg:p-10">
+		<CardSection className="max-w-xs">
+			<Image className="h-full w-full rounded-lg" src={imgUrl} alt="Contact Img" />
```

Comment:
`import NextImage from "next/image";`

```suggestion
			<Image
				component={NextImage}
				className="h-full w-full rounded-lg"
				src={imgUrl}
				alt="Stamford Logo"
				width={384}
				height={216}
			/>
```

Reply:
There are better ways to size this dynamically aswell, but I really can't be bothered looking it up right now, you can also use `fill`, but that will also require [sizes](https://nextjs.org/docs/pages/api-reference/components/image#sizes)